ui/tests/acceptance/replication/dr/secondary-test.js

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { later } from '@ember/runloop';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentURL, visit, pauseTest } from '@ember/test-helpers';
+
+import authPage from 'vault/tests/pages/auth';
+
+module('Acceptance | DR secondary details', function(hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(function() {
+    return authPage.login();
+  });
+
+  test('visiting DR Recovery Details page', async function(assert) {
```

Comment:
total nit, but this should be in `it [does a thing]` format ~ something like "it visits the Disaster Recovery Details page".

Reply:
I always forget this one.  Thanks for catching it!